Sources/Testing/Parameterization/Test.Case.Generator.swift

```diff
@@ -257,7 +257,30 @@ extension Test.Case {
 
 extension Test.Case.Generator: Sequence {
   func makeIterator() -> some IteratorProtocol<Test.Case> {
-    _sequence.lazy.map(_mapElement).makeIterator()
+    sequence(state: (
```

Comment:
I would recommend pulling the state structure out of the arguments list here so it's a bit easier to read. Something like:

```swift
let state = (
  ...
)
return sequence(state: state) { state in
  ...
}
```